tests/models/gpt_sw3/test_tokenization_gpt_sw3.py

```diff
@@ -128,3 +128,27 @@ def test_tokenizer_integration(self):
             model_name="AI-Sweden/gpt-sw3-126m",
             sequences=sequences,
         )
+
+    def test_build_conversation_input_ids_single_response(self):
+        tokenizer = self.get_tokenizer()
+        conversation = Conversation("Hvem er du?")
+        conversation_input_ids = tokenizer._build_conversation_input_ids(conversation=conversation)
+        self.assertEqual(
+            conversation_input_ids,
+            tokenizer.encode("<|endoftext|><s>\nUser:\nHvem er du?\nBot:\n"),
+        )
+
+    def test_build_conversation_input_ids_multiple_responses(self):
+        tokenizer = self.get_tokenizer()
+        conversation = Conversation()
+        conversation.add_user_input("Hvem er du?")
+        conversation.append_response("Jeg er en bot.")
+        conversation.add_user_input("Hvad hedder du?")
+        conversation_input_ids = tokenizer._build_conversation_input_ids(conversation=conversation)
+        self.assertEqual(
+            conversation_input_ids,
+            tokenizer.encode(
+                "<|endoftext|><s>\nUser:\nHvem er du?\nBot:\nJeg er en bot.\n"
+                "<s>\nUser:\nHvad hedder du?\nBot:\n",
+            )
+        )
```

Comment:
👍🏻 On @amyeroberts' comment, the test  that we usually implement is: 
```python 
    @slow
    def test_sequence_builders(self):
        tokenizer = self.tokenizer_class.from_pretrained("yjernite/retribert-base-uncased")

        text = tokenizer.encode("sequence builders", add_special_tokens=False)
        text_2 = tokenizer.encode("multi-sequence build", add_special_tokens=False)

        encoded_sentence = tokenizer.build_inputs_with_special_tokens(text)
        encoded_pair = tokenizer.build_inputs_with_special_tokens(text, text_2)

        assert encoded_sentence == [101] + text + [102]
        assert encoded_pair == [101] + text + [102] + text_2 + [102]
```
Which does not call the pipeline

Reply:
Hi @ArthurZucker. That exact test already exists, and it also seems to me that it's only testing the basic tokenization of the model, and not the chat format as implemented in the `_build_conversation_input_ids` method. But I just searched through the test modules and none of the other tokenizers with `_build_conversation_input_ids` have any tests associated with it, so I guess I can just remove them here?